mlflow/xgboost/__init__.py

```diff
@@ -345,7 +346,16 @@ class _XGBModelWrapper:
     def __init__(self, xgb_model):
         self.xgb_model = xgb_model
 
-    def predict(self, dataframe):
+    def predict(self, dataframe, params: Optional[Dict[str, Any]] = None):
+        """
+        :param dataframe: Model input data.
+        :param params: Additional parameters to pass to the model for inference.
+
+                       .. Note:: Experimental: This parameter may change or be removed in a future
+                                               release without warning.
+
+        :return: Model predictions.
```

Comment:
We might not need this because this is a private class.

Reply:
Then seems all predict function within wrappers are not exposed to customers, should we remove the docstring for all of them?

Reply:
I think it's okay to have the docstrings there, no need to remove